contracts/implementations/streamr/VehicleStream.sol

```diff
@@ -62,7 +74,121 @@ contract VehicleStream is AccessControlInternal {
             )
         );
 
-        emit VehicleStreamAssociated(streamId, vehicleId);
+        string memory oldStreamId = vs.streams[vehicleId];
+        if (bytes(oldStreamId).length != 0) {
+            emit VehicleStreamDissociated(vehicleId, oldStreamId);
```

Comment:
@zer0stars Do we want to let people use the "default" stream if they already have another one set?

Reply:
@zer0stars Giving this a bump.

Reply:
Not necessary.I would throw an error if they have one set. Then have them disassociate and reattach.